utils/generic.py

```diff
@@ -127,24 +128,38 @@ def filter_low_conf_regions(mt: hl.MatrixTable, filter_lcr: bool = True, filter_
     :return: MT with low confidence regions removed
     :rtype: MatrixTable
     """
-    from gnomad_hail.resources import lcr_intervals_path, decoy_intervals_path, segdup_intervals_path
+    from gnomad_hail.resources import lcr_intervals_path, decoy_intervals_path, segdup_intervals_path, high_coverage_intervals_path
 
+    criteria = [True]
```

Comment:
not that it _really_ matters, but I'd rather have this be `[]` and then not trigger any filtering action if it stays empty